src/plotting.jl

```diff
@@ -3,7 +3,8 @@ interval_markers(L::Type{<:Bound}, R::Type{<:Bound}) = interval_marker.([L, R])
 interval_marker(x::Type{Closed}) = :vline
 interval_marker(x::Type{Open}) = :none
 
-@recipe function f(xs::AbstractVector{<:AbstractInterval{T}}, ys) where T
+# TODO: Add support for plotting unbounded intervals
```

Comment:
Open an issue and add to this comment?

Reply:
Yes, thanks. I did look into this and it got complicated quickly

Reply:
https://github.com/invenia/Intervals.jl/issues/110